src/components/ShippingAdress/ShippingAdress.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import './ShippingAdress.scss'
+import{Field} from 'react-final-form'
+
+interface IProps {
+    setFirstAdress:any;
+    setLastAdress: any;
+    validate: any;
+}
+
+const ShippingAdress:React.FC<IProps> = ({setFirstAdress, setLastAdress, validate}) => {
```

Comment:
ShippingAddress instead of ShippingAdress

Reply:
I think you have 'adress' typo in the whole document

Reply:
I fixed it to "address" as it's suppose to be.